.github/workflows/tests.yaml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['7.0', '7.1', '7.2', '7.3', '7.4', '8.0']
```

Comment:
Btw, what about bumping min PHP version? PHP 7.1 and lower are dangerous to use - https://www.php.net/supported-versions.php

Also, package will be still available for the older PHP versions with version 1.5, so there is no BC break